### ISSUE TITLE
Consider removing hard-coded shadowCopy settings in favor of config files.

### ISSUE THREAD

**Issue by Pilchie** [Pilchie/xunit.runner.wpf#31]

**Comment by jasonmalinowski** [comment]
Didn't @DustinCampbell fix this with #46?

**Comment by DustinCampbell** [comment]
Yes, I think so. Pilchie -- let me know if you were thinking something else.

**Comment by DustinCampbell** [comment]
Sorry, I didn't quite understand the issue. Even with specifying config files, we appear to still need this. At least, it doesn't seem like the Roslyn config files set this value.
